server/acl_casbin_policy_dba.csv

```diff
@@ -12,6 +12,7 @@ p, DBA, /project/{id}/repository, PATCH
 p, DBA, /project/{id}/repository, DELETE
 p, DBA, /project/{id}/deployment, GET
 p, DBA, /project/{id}/deployment, PATCH
+p, DBA, /project/{projectID}/sync/{roleProvider}, POST
```

Comment:
/project/{projectID}/member/sync

No need to add roleProvider since there is only one provider per project